src/views/AgmLocationChg.vue

```diff
@@ -411,7 +411,7 @@ export default class AgmLocationChg extends Mixins(CommonMixin, DateMixin, Filin
 
     // if required data isn't set, go back to dashboard
     if (!this.getIdentifier || this.filingId !== 0) {
-      this.$router.push({ name: Routes.DASHBOARD })
+      this.navigateToDashboard(this.getIdentifier)
```

Comment:
According to line 413, you might get here because the identifier is falsy (0 or null or undefined or '') and then you're trying to use the identifier in a navigation!

I don't think you'll ever get here if identifier is falsy, so I recommend just removing that check, here and in the other filings below.

In the worst case, if you get here then you'll try to navigate to the business dashboard with an invalid identifier and the business dashboard can try to handle that. This is the same scenario as the existing check so it's no worse.

Reply:
Updated

Reply:
Looks good.